coq-builtin.elpi

```diff
@@ -685,16 +685,29 @@ external pred coq.env.add-indt i:indt-decl, o:inductive.
 
 % Interactive module construction
 
-% [coq.env.begin-module Name ModTyPath] Starts a module, the modtype can be
-% omitted *E*
-external pred coq.env.begin-module i:id, i:option modtypath.
+% [coq.env.begin-module-functor Name ModTyPath FunctorArguments] Starts a
+% module, the modtype can be omitted *E*
+external pred coq.env.begin-module-functor i:id, i:option modtypath, 
+                                           i:list (pair id modtypath).
+
+%
```

Comment:
you want to add a `pred ...` line to silence a warning

Reply:
Thanks. The warnings should be fixed now.